tests/integration/postgres.py

```diff
@@ -124,6 +125,13 @@ async def _wait_for_postgres_server(
         time.sleep(interval_s)
 
 
+async def _init_postgres_server(postgres_dsn: str) -> None:
+    if postgres_dsn.startswith("postgresql+asyncpg://"):
+        postgres_dsn = "postgresql" + postgres_dsn[len("postgresql+asyncpg") :]
+    conn = await asyncpg.connect(postgres_dsn)
+    await conn.execute("CREATE EXTENSION pgcrypto")
```

Comment:
should we have a dedicated migration for this?

Reply:
It should be handled by helm chart
https://github.com/neuro-inc/platform-api/pull/2116/files#diff-0e00ce289ad021751650ab6582b157daaec5f859b3341544f3fbbc55fbef24f2R43

Reply:
yeah, although a migration would work for both tests and production. but that's minor.